config/ks-core/templates/ks-apiserver.yml

```diff
@@ -43,12 +43,14 @@ spec:
         volumeMounts:
         - mountPath: /var/run/docker.sock
```

Comment:
I think we can remove `docker.sock` in v3.2, since there are no components depending on this.